dotCMS/src/main/java/com/dotcms/publishing/PublisherAPIImpl.java

```diff
@@ -84,6 +86,9 @@ public PublishStatus publish ( PublisherConfig config, PublishStatus status ) th
                 // If the bundle exists and we are retrying to push the bundle
                 // there is no need to run all the bundlers again.
                 if (!bundleExists || !publishAuditAPI.isPublishRetry(config.getId())) {
+                    PublishAuditHistory currentStatusHistory = publishAuditAPI
+                            .getPublishAuditStatus(config.getId()).getStatusPojo();
+                    currentStatusHistory.setBundleStart(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)